nixos/tests/matrix/mautrix-meta-sqlite.nix

```diff
@@ -0,0 +1,142 @@
+import ../make-test-python.nix ({ pkgs, ... }:
+  let
+    homeserverDomain = "server";
+    homeserverUrl = "http://server:8008";
+    userName = "alice";
+    botUserName = "facebookbot";
+  in
+  {
+    name = "mautrix-meta-sqlite";
+    meta.maintainers = pkgs.mautrix-meta.meta.maintainers;
+    meta.timeout = 300;
+
+    nodes = {
+      server = { config, pkgs, ... }: {
+        services.matrix-synapse = {
+          enable = true;
+          settings = {
+            database.name = "sqlite3";
+            app_service_config_files = [
+              config.services.mautrix-meta.instances.facebook.registrationFile
```

Comment:
Should the registration files be added automatically by the bridge configuration? Since also synapse is made to start only after the registration automatically, it would make sense.

Reply:
I think this is fine, but it would probably make sense to have a disable flag. Since all of the defaults are for localhost anyways and there is no good way to manage this cross-host I think enabling it by default makes sense.

Reply:
I am not sure I understand what you mean. This is from test, I am asking if to move into the module. Disable/enable for localhost is kind of automatic for the synapse wants etc., because it's included only if `services.mautrix-synapse.enable` is active

Reply:
Sorry. I meant it makes sense to move it into the module. 

> because it's included only if services.mautrix-synapse.enable is active

I would actually just reverse this. You can set the config option by default but it won't matter unless unless synapse is enabled. 

But there may be cases where people have synapse enabled but don't want this added automatically. So it would be nice to have an opt-out flag. (In fact I may use this as I am currently generating he registration myself. Although after the latest changes I can probably move to the automatic setup.)

Reply:
I've added `registerToSynapse` option and added the registration to Synapse when it's true